### PR TITLE
Small fix of `islice_extended` test

### PR DESCRIPTION
**Pull request by nanouasyn**
### Changes

Previously, the test checked the same `step` 4 twice, although the author obviously meant -4. I fixed it. Also, all `start` and `stop` values from -4 to 9 were checked, although it is enough to check the usual values, values at the borders, and values beyond the borders. I changed that too and this test was performed twice as fast. It ain't much but it's honest work.

### CODE REVIEW

File: tests/test_more.py [hunk 1]
```diff
@@ -3218,8 +3218,8 @@ def test_math(self):
 class IsliceExtendedTests(TestCase):
     def test_all(self):
         iterable = ['0', '1', '2', '3', '4', '5']
-        indexes = list(range(-4, len(iterable) + 4)) + [None]
-        steps = [1, 2, 3, 4, -1, -2, -3, 4]
+        indexes = [-10, -6, -5, -4, -2, -1, 0, 1, 2, 4, 5, 6, 10, None]
```
Comment (bbayles) [inline tests/test_more.py:3221]: Why are we not testing between `-3` and `3` now?

Comment (nanouasyn) [reply tests/test_more.py:3221]: It seems to me that `3` and `-3` are unremarkable. We are testing `2` and `-2`, also the usual values. `0` is the edge case, `-1` and `1` are close to `0`. `5` and `-5` are the edge cases. `4`, `-4`, `5` and `-5` are close to these cases. `10` and `-10` are far beyond the borders.

Comment (bbayles) [reply tests/test_more.py:3221]: I'll approve and merge this if we put those back; I meant for that whole range to be tested. Missing `-4` was definitely a mistake, though!

**Review by bbayles — APPROVED** [review]
